debug-ng-update.tsx

```
@@ -0,0 +1,43 @@
+    });
+    
+    // 既存のロジック...
+    const result = [];
+    
+    console.log('[displayItems] Result:', result.length, 'items');
+    return result;
+  }, [itemsWithTags, config.tag, filterItems, ngList]);
+  
+  // filterItems関数もログを追加
+  const filterItems = useCallback((items) => {
+    console.log('[filterItems] Called with', items.length, 'items');
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)